src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, sex, isMarried, partnerName, age } = person;
+  let status = 'I am not married';
+
+  if (isMarried) {
+    if (sex === 'f') {
+      status = `${partnerName} is my husband`;
+    } else {
+      status = `${partnerName} is my wife`;
+    }
```

Comment:
```suggestion
    status = `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`;
```
Could be simplified, as only husband/wife word changes